src/client/theme-default/components/NextAndPrevLinks.ts

```diff
@@ -1,13 +1,43 @@
 import { defineComponent, computed } from 'vue'
 import { usePageData } from 'vitepress'
+import { useSiteData } from 'vitepress'
```

Comment:
Can we merge this one with the above import?

```suggestion
import { usePageData, useSiteData } from 'vitepress'
```

Reply:
Thank you!
Merged!!